ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -218,17 +218,86 @@ class SignalProducerSpec: QuickSpec {
 			}
 		}
 
+		describe("ignoreValues") {
+			it("should not forward values from original producer") {
+				let producer: SignalProducer<Int, NoError> = SignalProducer<Int, NoError>(values: [ 1, 2 ]) |> ignoreValues
```

Comment:
The typing here is unfortunate. Is there a way to have it to use the obvious default?